bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -310,8 +334,11 @@ func (s *ksServer) CreateApp(ctx context.Context, request CreateRequest) error {
 	} else {
 		log.Infof("Creating app %v", request.Name)
 		log.Infof("Using K8s host %v", config.Host)
-
-		appDir := path.Join(s.appsDir, GetRepoName(request.Project), request.Name)
+		deployConfDir := path.Join(s.appsDir, repoDir, GetRepoName(request.Project), kfVersion, request.Name)
```

Comment:
Can we do a defer delete here?

Reply:
Now should be covered by https://github.com/kubeflow/kubeflow/pull/1893/files#diff-98496810c3874df46e02c51893aa04c1R318?